src/telemetry/metrics.yaml

```diff
@@ -0,0 +1,30 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# For more information on configuring this file:
+# https://mozilla.github.io/glean/book/reference/yaml/metrics.html
+
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+app_events:
+  page_view:
+    type: event
+    description: |
+      A generic page view event with extras.
+    bugs:
+      # TODO: Add bug link
+      - https://bugzilla.mozilla.org/1691365
+    data_reviews:
+      # TODO: Add data review link
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1691365#c0
+    data_sensitivity:
+      - technical
```

Comment:
Doesn't matter too much, but this is not "technical", it's "interaction" (see https://wiki.mozilla.org/Data_Collection)